src/SFA.DAS.ProviderApprenticeshipsService.Web/Helpers/FeatureEnabledCheck.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+using System.Threading.Tasks;
+using System.Web.Mvc;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.Helpers
+{
+    public static class FeatureEnabledCheck
+    {
+        public static bool IsManageReservationsEnabled(this HtmlHelper htmlHelper, long providerId)
+        {
+            var service = DependencyResolver.Current
+                .GetService<IFeatureToggleService>();
+            var isEnabled = service.Get<Domain.Models.FeatureToggles.ManageReservations>().FeatureEnabled;
```

Comment:
could Domain.Models.FeatureToggles be put in as a namespace. Then it would read as 

 service.Get<ManageReservations>) 

which looks better